src/sass/component/_content-area.scss

```diff
@@ -0,0 +1,20 @@
+main {
```

Comment:
This also seems like a lot of code. Not sure why side-nav is display none. There is also some janky spacing and indentation.